packages/design-system/src/docs/Themes.tsx

```diff
@@ -132,7 +129,8 @@ const SAliasCategory = styled.article``;
 
 const previewWidth = 40;
 
-const ColorToken = ({ name, theme }) => {
+const ColorToken = ({ name, theme }: { name: string; theme: DefaultTheme }) => {
+	// @ts-ignore
```

Comment:
```suggestion
	// @ts-ignore
```
Why? It'd be great to get into the habit of explaining the ts ignores with a comment

Reply:
Fixed